wooCommerce_scripts/Scripts/create_products.py

```diff
@@ -0,0 +1,39 @@
+import random
+import string
+from WooCommerceUtility import WooCommerceUtility
+
+
+def generate_random_string(length=10, prefix=None, suffix=None):
+    random_string = ''.join(random.choices(string.ascii_lowercase, k=length))
+    if prefix:
+        random_string = prefix + random_string
+    if suffix:
+        random_string = random_string + suffix
+    return random_string
+
+
+def create_100_simple_products():
+    # Set up WooCommerce API client
+    woo_helper = WooCommerceUtility()
+
+    for i in range(1, 101):
+        # Generate data for each product
+        payload = {
+            "name": generate_random_string(),
+            "type": "simple",
+            "regular_price": "10.99"
+        }
+
+        # Make API call to create product
+        products_rs = woo_helper.wcapi.post("products", payload)
```

Comment:
ERROR
why are you making 'woo_helper.wcapi'?
The 'woo_helper' object it self has the 'post' function. You created the 'post' function so that you don't have to use the wcapi directly and you have more validation like status_code validation. The way you called it, what would happen if the status code is bad? Ask your self why did you create the 'post()' function? Are u even using it here?
This part is very important. I hope you understand it. If not lets chat on Zoom and let me explain.
So you should be doing
`products_rs = woo_helper.post("products", params=payload)`